plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPlugin.kt

```diff
@@ -356,7 +356,7 @@ private fun Project.createAndroidVariantMetaKtlintCheckTask(
     variantName: String,
     multiplatformTargetName: String? = null
 ): TaskProvider<Task> = registerTask(variantName.androidVariantMetaCheckTaskName(multiplatformTargetName)) {
-    group = VERIFICATION_GROUP
+    group = KTLINT_GROUP
```

Comment:
Initial discussion was to change groupless tasks to have "Verification" group, though in this PR you've changed tasks that already had "Verification" group to have "ktlint" group. Why so?

Reply:
Ah I was going off of the later comment o make them ktlint due to the volume and noise it causes